src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/RazorCompletionEndpoint.cs

```diff
@@ -229,18 +229,19 @@ public Task<CompletionItem> Handle(CompletionItem completionItem, CancellationTo
                 completionItem = completionItem with { Documentation = documentation };
             }
 
+            // We might strip out the commitcharacters for speed, bring them back
+            var container = associatedRazorCompletion.CommitCharacters != null ? new Container<string>(associatedRazorCompletion.CommitCharacters) : null;
+            completionItem = completionItem with { CommitCharacters = container };
+            var vsCompletionItem = completionItem.ToVSCompletionItem(_capability?.VSCompletionList);
+
             if (tagHelperClassifiedTextTooltip != null)
             {
-                // We might strip out the commitcharacters for speed, bring them back
-                var container = associatedRazorCompletion.CommitCharacters != null ? new Container<string>(associatedRazorCompletion.CommitCharacters) : null;
-                var withCommit = completionItem with { CommitCharacters = container };
-                var vsCompletionItem = withCommit.ToVSCompletionItem(_capability?.VSCompletionList);
                 completionItem = completionItem with { Documentation = string.Empty };
```

Comment:
I think this was an oversight from when this code was originally included. We can remove